tests/unit/test.test.ts

```
@@ -0,0 +1,8 @@
+describe('example test suite', () => {
+  it('should pass a basic assertion', () => {
+    const result = 2 + 2;
+    expect(result).toBe(4);
+  });
+});
+
+export {};
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)